### DISCUSSION TITLE
Setting header again appends to header, not replaces

### DISCUSSION THREAD

**Discussion by dalisoft** [uNetworking/uWebSockets.js#151] [Q&A] [answered]
Hi.

Related issue: #150 

As title says, when again setting `res.setHeader(key, value)` appends to same header, not replaces.

It is bug? Or am i doing something wrong?

**Comment by ghost** [comment, marked as answer]
There is no res.setHeader, you only have res.writeHeader

**Comment by dalisoft** [comment]
Yep, setHeader in my app :)
I forgot to say, yes, same bug applies to writeStatus too
